### DISCUSSION TITLE
Last seen of “N/A” for everything

### DISCUSSION THREAD

**Discussion by SadGamerGeek** [Koenkk/zigbee2mqtt#8102] [General]
New user here, so I’m probably missing something obvious.

I have a mix of 7 devices setup so far, on slaesh’s stick. They’ve been connected for a couple of days, but none of them have a valid “Last Seen” date/time - they all show “N/A”. I found an earlier issue logged for this but that was fixed some versions ago. Is this an issue, or just configuration?
![6A5E34E1-6667-4C5F-95C9-8B27CAB97793](https://user-images.githubusercontent.com/13276467/126564947-838673d8-b382-4854-8a32-632b0dbed37d.jpeg)

 Also, in that issue relating to an older version, there was a “last seen” column in the devices list, whereas I don’t have that. Is it a config option somewhere?

![FF1F23BC-E939-4A7B-8823-D90BFF6AF718](https://user-images.githubusercontent.com/13276467/126564955-b0b351f1-9aa7-4dc8-a2ca-bb1dfa5c4ad2.jpeg)

**Comment by Koenkk** [comment]
You probably need to enable the `last_seen` option first (https://www.zigbee2mqtt.io/information/configuration.html)

**Reply by SadGamerGeek** [reply]
Perfect! Working well now, after adding this as “ISO_8601_local”

![3D2760BD-29C8-4F84-B28B-F9D645C1107A](https://user-images.githubusercontent.com/13276467/126629697-47908c80-0c86-428f-9c54-e0b86e0052b3.jpeg)

Apologies for my failure to “RTFM”! Will properly check those options next time…
